client/ayon_core/hosts/hiero/api/otio/hiero_import.py

```diff
@@ -397,28 +396,21 @@ def create_trackitem(playhead, track, otio_clip, clip):
         source_in = source_range.end_time_inclusive().value
 
         timeline_in = playhead + source_out
-        timeline_out = (
-            timeline_in +
-            source_range.duration.value
-        ) - 1
+        timeline_out = (timeline_in + source_range.duration.value) - 1
     else:
         # Normal playback speed
         source_in = source_range.start_time.value
         source_out = source_range.end_time_inclusive().value
 
         timeline_in = playhead
-        timeline_out = (
-            timeline_in +
-            source_range.duration.value
-        ) - 1
+        timeline_out = (timeline_in + source_range.duration.value) - 1
```

Comment:
Do we still care about keeping the brackets?
```suggestion
        timeline_out = timeline_in + source_range.duration.value - 1
```

Reply:
Technically the same, but visually helps to keep it in brackets.

Reply:
Cool - feel free to mark resolved.